### ISSUE TITLE
pre-trained model?

### ISSUE THREAD

**Issue by rasunju** [mmlab-cv/MoMa#2]
Thanks for your great work, is there a pre-trained model?

**Comment by giuli13** [comment]
Yes! You can find the checkpoint at the following [link](https://drive.google.com/drive/folders/1OE7-mgmSDZizNHU2-CjvWBlq7Hy3P866?usp=sharing)

**Comment by rasunju** [comment]
excellent! thx so much.
